config/longrun_configs/longrun_aquaplanet_rhoe_equil_55km_nz63_clearsky_0M_earth.yml

```diff
@@ -10,8 +10,8 @@ z_max: 55000.0
 rayleigh_sponge: true
 topography: "Earth"
 topo_smoothing: true
-kappa_4_vorticity: 4.0
-kappa_4_scalar: 4.0
+vector_hyperdiffusion_coefficient: 4.0
+scalar_hyperdiffusion_coefficient: 4.0
```

Comment:
@szy21 how about these as names? they seem a bit more descriptive

Reply:
This looks better I think. But maybe `vorticity_hyperdiffusion_coefficient` instead of `vector_hyperdiffusion_coefficient` because the grad div term in momentum hyperdiffusion is also a vector but uses vorticity_hyperdiffusion_coefficient * divergence damping factor.

In the code I think it would be more clear to name them something like `ν₄_vorticity_coeff' and `ν₄_vorticity`, instead of `ν₄` being the coefficient and `κ₄` being the actual value.